release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -424,6 +424,12 @@ The Network Observability Operator releases updates independently from the {prod
 //[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
 //=== Cloud controller managers for additional cloud providers
 
+[discrete]
+[id="ocp-4-15-ports-use-tls"]
+=== Cluster metrics ports secured
+
+With this release, the ports that serve metrics for the cluster machine approver and cloud controller manager Operators use the Transport Layer Security (TLS) protocol for additional security. (link:https://issues.redhat.com/browse/OCPCLOUD-2272[*OCPCLOUD-2272*], link:https://issues.redhat.com/browse/OCPCLOUD-2271[*OCPCLOUD-2271*])
```

Comment:
Minor comment, but I'm curious whether "cloud controller manager Operator" etc need any capitalization or an attribute added? Probably out of scope for this PR though.